src/System.Windows/ref/System.Windows.csproj

```diff
@@ -1,15 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <PackageTargetFramework>net46</PackageTargetFramework>
     <NuGetTargetMoniker>.NETFramework,Version=v4.6</NuGetTargetMoniker>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <TargetingPackReference Include="System.ComponentModel.Composition" />
+    <Compile Include="System.Windows.cs" />
```

Comment:
you should be able to sanity check at the end by running genapi on what the build produces, and it should show this omission as a diff?

---
In reply to: [68113617](https://github.com/dotnet/corefx/pull/9585#discussion_r68113617) [](ancestors = 68113617,68095665)